Readme.md

```diff
@@ -16,38 +16,44 @@ A simple app to practice Hebrew verbs.
 
 ## Configuration
```

Comment:
It has been a while since I last dockerized applications, but at that time the advice was not to put your database in a database because you would lose your data when recreating the container. If that is still the case, I would suggest either adding the docker-compose instructions under a separate heading for development purposes, or alternatively explaining how to set up a volume to persist the data. I have a configuration [here](https://gitlab.com/cloogle/cloogle-web/-/blob/main/docker-compose.yml?ref_type=heads) that does this (I don't know if it follows modern best practices, but if it works here that is not a big issue to me).

Reply:
Yes, containers do not have persistent storage. However one can get around this using volumes. The `mysql` container by default creates an unnamed volume to persist its data (see https://hub.docker.com/_/mysql under "Where to Store Data") and this is the system currently being used. If you'd like, I'd happily switch to creating a named volume, but I don't see a significant advantage to it at the moment.

Reply:
That still says:

> The downside is that the files may be hard to locate for tools and applications that run directly on the host system, i.e. outside containers.

It is not clear to me how to recover data after you have deleted the container by accident, for example. This is more transparent when the volume is mounted to a path in the repository. If it's the same to you, I'd prefer this method (you can make sure that the user doesn't have to create the directory by adding an empty `.gitkeep` file in the directory).